### ISSUE TITLE
random crash on calling 'rollnext' on pseudofullscreen tiled window

### ISSUE THREAD

**Issue by Seme4eg** [hyprwm/Hyprland#4342]
### Hyprland Version

Hyprland, built from branch main at commit 1512b81126dd115f089fd21244692d92034c78f8 dirty (master: guard PNODE in roll*).
Date: Wed Jan 3 00:18:38 2024
Tag: v0.34.0-19-g1512b811

flags: (if any)

### Bug or Regression?

Bug

### Description

tried to rollnext to 2nd window in workspace, thing hung and compositor crashed. 

### How to reproduce

master layout, pseudofullscreen windows, 2 windows on workspace, trying to call rollnext (i also have layoutmsg:cyclenext bound to same keybinding, so they both are called).

### Crash reports, logs, images, videos

[hyprlandCrashReport1146.txt](https://github.com/hyprwm/Hyprland/files/13823263/hyprlandCrashReport1146.txt)

**Comment by thejch** [comment]
Can't reproduce
By pseudofullscreen do you mean maximize

**Comment by Seme4eg** [comment]
yes, i mean window fullscreened with `bind=SUPERCTRL,F,fullscreen,1 # maximize (gaps left)`

**Comment by Seme4eg** [comment]
and yes, for me it happened only once, i thought that crashreport can be informative. If same will happen to me again i will attach second report.
